client/blocks/plugins-update-manager/index.tsx

```diff
@@ -68,7 +70,7 @@ export const PluginsUpdateManager = ( props: Props ) => {
 							onClick={ onCreateNewSchedule }
 							disabled={ ! canCreateSchedules }
 						>
-							Create a new schedule
+							{ translate( 'Set up a new schedule' ) }
```

Comment:
Can we simplify this to "Create new schedule"? Or "Add new schedule" to align it with "Add new post"?

Reply:
It used to be "Create a new schedule", but the copy was reviewed and changed to be "Set up a new schedule".

@JanaMW27, what do you think?

Reply:
Both copies sound good to me. Maybe "set up" is more aligned with the user action? I dunno, both work for me.